base/libgit2/commit.jl

```diff
@@ -22,6 +22,12 @@ function committer(c::GitCommit)
     return Signature(ptr)
 end
 
+function Base.show(io::IO, c::GitCommit)
+    authstr = sprint(show, author(c))
+    cmtrstr = sprint(show, committer(c))
+    print(io, "Commit Author: $authstr\nCommitter: $cmtrstr\nSHA: $(Oid(c))\nMessage:\n$(message(c))")
```

Comment:
Also, it would probably be useful to print the typename, both here and the other PRs.